context/app/static/js/pages/Genes/Genes.jsx

```diff
@@ -17,15 +21,19 @@ function Genes() {
     .map(([sectionName]) => sectionName);
   const sections = new Map(getSections(sectionOrder));
 
+  if (isLoading) {
+    return null;
+  }
+
   return (
     <FlexRow>
       <TableOfContents items={[...sections.values()]} />
       <Content>
-        <Typography variant="subtitle1" component="h1" color="primary">
+        <Typography variant="subtitle1" component="h2" color="primary">
           Gene
         </Typography>
         <Typography variant="h1" component="h2">
```

Comment:
One final tiny fix to go along with the revision to use `h2` for the `gene` subtitle:

```suggestion
        <Typography variant="h1" component="h1">
```

Reply:
great catch!